### PR TITLE
Add a bit more documentation

### PR DESCRIPTION
**Pull request by Nek-**
This PR adds more "TODO" (which is pretty bad :>). But also add a new « usage » section that explain how woketo work.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,11 +25,65 @@ Requirements
...
+<?php
+// MessageHandler.php
+
+namespace Your\Namespace;
+
+use Nekland\Woketo\Server\Connection;
+
+class YourMessageHandler implements \Nekland\Woketo\Message\MessageHandlerInterface
+{
+    public function onConnection(\Nekland\Woketo\Server\Connection $connection)
```
Comment (valanz) [inline README.md]: `Connection` instead :)

File: README.md [hunk 2]
```diff
@@ -22,11 +25,65 @@ Requirements
 How to install
 --------------
 
+*Woketo is not on composer for now because it's not released. If you really want to use it you can add it as git dependency.*
```
Comment (folliked) [inline README.md]: add "composer install ", even if it is obvious

**Comment by Nek-** [comment]
Require does it.

File: README.md (hunk 2 above)
Comment (Nek-) [inline README.md]: `composer require` actually do the job.

**Pull request merged**